puppet-checks/src/main/resources/org/sonar/l10n/pp/rules/puppet/UnquotedResourceTitle.html

```diff
@@ -20,9 +25,13 @@
   ensure => directory,
 }
 
+file { ${bar}:
+  ensure => directory,
+}
+
 file { [
     'abc',
-    "${var}",
+    ${var},
```

Comment:
@petems My bad... Thanks a ton for your feedback! Here's what I meant: https://github.com/iwarapter/sonar-puppet/pull/261